app/src/gui/components/record/relationships/create_links/create_record_link.tsx

```diff
@@ -47,12 +47,12 @@ export function AddNewRecordButton(props: {
         .handleSubmit()
         .then((revisionID: RevisionID) => {
           const newState = props.state;
```

Comment:
So if handleSubmit fails we'll still spin, maybe we could pop up an error alert in this case just so the UI still responds.  It would be the un-useful 'Error saving record' alert that we use somewhere in the save method.